### DISCUSSION TITLE
Question: Reporting issues for the V2 doc

### DISCUSSION THREAD

**Discussion by hadware** [litestar-org/litestar#1413] [Q&A] [answered]
### Summary

I'm trying to use starlite v2.0.0alpha2 (for a personal project, i'm ok with the alpha status). I'm trying to migrate my v1 code to v2, but some stuff isn't properly documented yet in the V2 doc. What's the best way for me to report these problems, as to help you as much as possible in fixing them?

Should I create a single issue for each single tiny problem I encounter in the doc? Should I group them?

**Comment by provinzkraut** [comment, marked as answer]
> Should I create a single issue for each single tiny problem I encounter in the doc? Should I group them?

Well that depends. If they're completely unrelated it's probably best to open a separate issue, but if it's things within the same document / module, you can just open one issue for all of them.

You can of course also dump all your finding here and we'll sort out how best to track them :wink:

**Comment by hadware** [comment]
All right. Thanks for making this a discussion. I'll try to get some time this evening to list my findings in here.

**Comment by hadware** [comment]
Ok, here's a list of what i've found for now. This is completely unordered, and very much influenced by the scope of what I wanted to do with starlite.

- [ ] [the OpenAPI generation CLI doc](https://docs.starliteproject.dev/2/usage/cli.html#openapi) isn't up to date with the current CLI: the new command is `starlite schema typescript` (for instance, to get the typescript client lib)
- [ ] [the tutorial on JWT Auth uses the old typing API for the `Request` type](https://docs.starliteproject.dev/2/usage/security/abstract-authentication-middleware.html#example-implementing-a-jwtauthenticationmiddleware). The V2 type is `Request[User,Token,State]`.

I'll add more as I find more to add.

**Comment by provinzkraut** [comment]
Okay, since we have a few more items, I created a tracking issue: #1421. Let's add new things there.

**Reply by hadware** [reply]
Perfect!
